apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -121,8 +121,32 @@ type BackendTLSPolicyValidation struct {
 	// not both. If CACertificateRefs is empty or unspecified, the configuration for
 	// WellKnownCACertificates MUST be honored instead if supported by the implementation.
 	//
-	// References to a resource in a different namespace are invalid for the
-	// moment, although we will revisit this in the future.
+	// A CACertificateRef can be invalid for the following reasons. In all cases, the
+	// implementation MUST ensure the `ResolvedRefs` Condition on the BackendTLSPolicy
+	// is set to `status: False`, with a Reason and Message that indicate the cause
+	// of the error. Connections using an invalid CACertificateRef MUST fail, and the
+	// client MUST receive an HTTP 5xx error response. If all CACertificateRefs are
+	// invalid, the implementation MUST also ensure the `Accepted` Condition on the
+	// BackendTLSPolicy is set to `status: False`, with a Reason `NoCACertificates`.
+	//
+	// A CACertificateRef is invalid if:
```

Comment:
I think the definition for invalid BTP should be before a block with Status definition.